out/blocks/wd_admin_payment_main_form.tpl

```diff
@@ -113,6 +145,11 @@
           [{$oViewConf->getInputHelpHtml($configField.description)}]
         [{/if}]
       </td>
+      [{if $configKey === 'countryCode'}]
```

Comment:
We seem to need validation result areas in multiple places now. It might be better to generate an empty `<td>` element next to every row so that we don't have to create these for specific field keys.

Reply:
Good point, will take care of it.

Reply:
The template is now refactored so that no checks for specific field IDs are necessary anymore.